docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Exciting MUI X improvements: v6.11.0
+description: Support for time zones, Charts in alpha, Data Grid filtering, and more.
+date: 2023-08-07T00:00:00.000Z
+authors: ['richbustos', 'josefreitas']
+card: true
+tags: ['MUI X', 'News']
+---
+
+<img src="/static/blog/mui-x-mid-v6-features/intro.png" alt="" width="2400" height="600" style="margin-bottom: 16px;" />
+
+Since the first v6 stable release, we've continuously rolled out new major features.
+Now that we've reached [MUI X v6.11.x](https://github.com/mui/mui-x/releases), it's time to delve into the most recent additions to MUI X v6, and why you should get the latest version now.
+
+## Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Date Pickers](#date-pickers)
+  - [Support for time zones](#support-for-time-zones)
+  - [Digital clock](#digital-clock)
+- [Data Grid](#data-grid)
+  - [Copy and paste](#copy-and-paste)
```

Comment:
We are missing the plan no?

```suggestion
  - [Copy and paste](#copy-and-paste) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
```

Reply:
I removed it and put it on the headers and not on the table of content.

Will add it back on